templates/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="{{ url_for('static', filename='img/favicon.ico') }}">
+    <link rel="stylesheet" href="{{ url_for('static', filename='/styles/index.css') }}">
+    <title>URL Shortener</title>
+</head>
+<body>
+    <h1>URL shortener</h1>
+    <form method="POST" action="{{ url_for('home') }}">
+        <input type="text" name="original_url">
+        <input type="submit">
+    </form>
+    <p >{{ url }}</p>
```

Comment:
At the moment this just shows the code, might be nice to show the entire link, e.g. http://localhost:5000/Dtg so that user can directly click on it.